examples/two_functions_with_dispatch.md

```diff
@@ -0,0 +1,187 @@
+
+## Full container bytecode
+
+```
+ef0001 030006 01003b 010017 01001d 00 000001010101 60043560e06000351c639fb890d581145d001c6320cb776181145d00065050600080fd50fb000260005260206000f350fb000160005260206000f3 600181115d0004506001fc60018103fb000281029050fc 600281115d0004506001fc60028103fb000160018203fb0001019050fc
+```
+
+## Dispatch function
+
+### Pseudocode
+
+```
+function main()
+{
+    let i := shr(calldataload(4))
+    let sig := shr(calldataload(0), 224)
+    if (eq(sig, 0x9fb890d5))
```

Comment:
Is this `fac(uint256)`?

Reply:
`fact(uint)`, I'll add a comment

Reply:
In ABI can't use type shorthand, so must be `fact(uint256)` then.